Rasa_Bot/actions/actions.py

```diff
@@ -135,3 +137,22 @@ async def run(self, dispatcher, tracker, domain):
         )
 
         return [reminder]
+
+
+class ActionSavePaEvaluationToDB(Action):
+    """"To save user input from PA evaluation form to database"""
+
+    def name(self):
+        return "action_save_pa_evaluation_to_db"
+
+    async def run(self, dispatcher, tracker, domain):
+
+        pa_evaluation_response = tracker.get_slot("pa_evaluation_response")
+
+        session = get_db_session()  # Creat session object to connect db
+
+        user_id = 1  # TODO_b take this value from database
+        selected = session.query(Users).filter_by(nicedayuid=user_id).one()
+        selected.PA_evaluation = pa_evaluation_response
```

Comment:
Please update this based on my comments in https://github.com/PerfectFit-project/virtual-coach-db/pull/2